internal/executor/docker.go

```diff
@@ -264,28 +266,40 @@ func (e *docker) Wait() error {
 
 	containerID := e.getContainerID()
 
+	handleContextError := func() (handled bool) {
```

Comment:
I thought we don't do named return values?

Reply:
I mean, I do them when the meaning is non-obvious what the return value is. I think the "no named return values" is not a great rule and should only be considered if you're using named returns values as a means to declare output variables. Otherwise, it's a convenient documentation tool (just like naming your input parameters).